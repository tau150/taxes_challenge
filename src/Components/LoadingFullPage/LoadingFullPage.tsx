import { VStack, Spinner, Heading, Box } from "@chakra-ui/react";

const LoadingFullPage = () => {
  return (
    <Box h="100vh" left="0" position="absolute" right="0" top="0" zIndex="-1">
      <VStack h="100%" justify="center">
        <Spinner color="green.300" size="xl" />
        <Heading color="green.300" mt="4" size="lg">
          Loading...
        </Heading>
      </VStack>
    </Box>
  );
};

export default LoadingFullPage;
