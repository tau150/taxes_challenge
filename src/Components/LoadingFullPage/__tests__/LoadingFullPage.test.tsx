import { expect } from "vitest";

import LoadingFullPage from "../LoadingFullPage";

import { render } from "tests/render";

describe("LoadingFullPage component", () => {
  it("should match with snapshot", () => {
    const { container } = render(<LoadingFullPage />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
