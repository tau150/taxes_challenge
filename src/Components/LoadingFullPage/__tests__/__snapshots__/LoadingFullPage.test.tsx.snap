// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoadingFullPage component > should match with snapshot 1`] = ` `;
